ext/dom/tests/modern/token_list/toggle.phpt

```diff
@@ -0,0 +1,80 @@
+--TEST--
+TokenList: toggle
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+
+$dom = DOM\XMLDocument::createFromString('<root class="A B C"/>');
+$element = $dom->documentElement;
+$list = $element->classList;
+
+echo "--- Toggle A (forced add) ---\n";
+
+var_dump($list->toggle("A", true));
```

Comment:
Considering the first echo says a forced add, don't you want the initial document to not have `A` in the class list?

Reply:
I explicitly want to test the behaviour of doing a forced add (i.e. $force=true) with a class already in the class list.

Reply:
ACK